variable/include/scipp/variable/data_model.h

```diff
@@ -14,12 +14,12 @@
 
 namespace scipp::variable {
 
-template <class T, class C> auto &requireT(C &concept) {
-  if (concept.dtype() != dtype<typename T::value_type>)
+template <class T, class C> auto &requireT(C &scippconcept) {
+  if (scippconcept.dtype() != dtype<typename T::value_type>)
```

Comment:
Maybe use the name `varconcept` here are well to make it consistent with the changes below?